### ISSUE TITLE
I found a critical bug in the loss function

### ISSUE THREAD

**Issue by jind11** [zw-s-github/MUDE#3]
hi, I found a very critical bug in your code with respect to the loss function: the output of your models is using softmax function and you used the NLLLoss function, which misses the log function. So the loss is now negative, which is wrong. I suggest replacing the F.softmax with F.log_softmax. Please check it. Thanks!
